### PR TITLE
Add sciplot package for R

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
Waiting for suggestions [here](https://github.com/conda-forge/staged-recipes/issues/3452)

**Comment by sebastian-luna-valero** [comment]
Please @conda-forge/staged-recipes review.

File: recipes/r-sciplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  fn: sciplot_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/sciplot_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/sciplot/sciplot_{{ version }}.tar.gz
+
+
+  sha256: c31b59d9cb16c8af5f054e51bd8cd37599886dc4cfc4a14f1f1f9d0367eabcf9
+  # patches:
+   # List any patch files here
+   # - fix.patch
```
Comment (scopatz) [inline recipes/r-sciplot/meta.yaml]: Please remove extra commented out lines in this file.

File: recipes/r-sciplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+# Title: Scientific Graphing Functions for Factorial Designs
+# Author: Manuel Morales <mmorales@williams.edu>, with code developed by the R Development Core Team and with general advice from the R-help listserv community, especially Duncan Murdoch.
+# Maintainer: Manuel Morales <Manuel.A.Morales@williams.edu>
+# Depends: stats
+# Description: A collection of functions that creates graphs with error bars for data collected from one-way or higher factorial designs.
+# License: GPL (>= 2)
+# NeedsCompilation: no
+# Packaged: 2017-06-19 21:49:09 UTC; mmorales
+# Repository: CRAN
+# Date/Publication: 2017-06-19 23:20:32 UTC
```
Comment (scopatz) [inline recipes/r-sciplot/meta.yaml]: These you can keep since they are informational.

**Comment by sebastian-luna-valero** [comment]
Close in favour of https://github.com/conda-forge/staged-recipes/pull/3872 ?

**Comment by sebastian-luna-valero** [comment]
Closing in favour of https://github.com/conda-forge/staged-recipes/pull/3872

**Pull request closed without merging**